pkg/client/transport/cache.go

```diff
@@ -22,6 +22,8 @@ import (
 	"net/http"
 	"sync"
 	"time"
+
+	utilnet "k8s.io/kubernetes/pkg/util/net"
```

Comment:
I'm in favor of this, but it was apparently contentious when this package was created. Would like agreement upstream if possible